File: src/test/java/com/alibaba/ttl/threadpool/agent/check/AgentCheckMain.java
package com.alibaba.ttl.threadpool.agent.check;

public class AgentCheckMain {
    public static void main(String[] args) {
        try {
            ExecutorClassesAgentCheck.main(args);
            ForkJoinTaskClassAgentCheck.main(args);
        } catch (Throwable e) {
            System.out.println("Exception when run AgentCheck: ");
            e.printStackTrace(System.out);
            System.exit(2);
        }
    }
}

File: src/test/java/com/alibaba/ttl/threadpool/agent/check/AgentCheckMain.kt
@file:JvmName("AgentCheckMain")

package com.alibaba.ttl.threadpool.agent.check

import com.alibaba.support.junit.conditional.BelowJava7

fun main(args: Array<String>) {
    com.alibaba.ttl.threadpool.agent.check.executor.main(args)

    if (!BelowJava7().isSatisfied)
        com.alibaba.ttl.threadpool.agent.check.forkjoin.main(args)
}
